e2e/test/scenarios/models/models-list-view.cy.spec.js

```diff
@@ -87,15 +87,19 @@ describe("scenarios > models list view", () => {
       cy.get("@listPreview").should("be.visible");
       cy.get("@listPreview").within(() => {
         // Verify preview shows sample data values
-        cy.findByRole("img").should("have.attr", "aria-label", "document icon");
+        cy.findAllByRole("img").should(
+          "have.attr",
```

Comment:
_comment in unrelated place_

When I click a header to sort the list, the header disappears (inconsistent with normal table view) while the request is pending.